extras/videoDrivers/WinAPI/WinAPI.cpp

```diff
@@ -443,6 +444,14 @@ bool VideoWinAPI::MessageLoop()
     return true;
 }
 
+/**
+ *  grafische Fehlermeldung
+ */
+void VideoWinAPI::ShowErrorMessage(const char* title, const char* message)
+{
+    MessageBox(NULL, (LPCWSTR)message, (LPCWSTR)title, MB_OK);
```

Comment:
You need to actually create a wide string here, not reinterpret-cast the char string to a wide string. And maybe directly use the wide API, see also https://github.com/Return-To-The-Roots/s25client/blob/544f85a329932db1d5bf3ebd7caea46d062930aa/libs/s25client/s25client.cpp#L132:

```suggestion
    MessageBoxW(nullptr, boost::nowide::widen(message).c_str(), boost::nowide::widen(title).c_str(), MB_OK);
```